src/main/resources/resource/layout/din1505/din1505winword.article.layout

```diff
@@ -7,10 +7,10 @@
 \end{volume}
  (\year)
 \begin{number}
-, Nr. \number
+, Nr. {\number}
```

Comment:
Here you don't need to add the curly braces, they are only needed when the there is a formatter (e.g HTMLChars) before.

Reply:
Ok, so just changing this line is supposed to work, right?
```
 "\\begin{journal}<BR><BR><i>\\format[HTMLChars]{\\journal}</i> \\end{journal} \\begin{volume}, Vol. \\volume\\end{volume}\\begin{series}<BR> \\format[HTMLChars]{\\series}\\end{series}\\begin{number}, No. \\format[HTMLChars]{\\number}\\end{number}__NEWLINE__" +
```

Reply:
Yes! For more information: https://docs.jabref.org/collaborative-work/export/customexports